### ISSUE TITLE
BZ Gas does not work

### ISSUE THREAD

**Issue by ItsTheSneakyTurtle** [Austin-R418/TARP-Bug-Tracker#8]
**Describe the bug**
So the police have access to BZ Gas however they are not usable whatsoever. They constantly mess up the inventory system and it is very hard to drag them around, for example: if I try to drag the BZ gas it will not move but then suddenly jumped to a random slot and switching places with the previous item there.

**To Reproduce**
1. Get BZ gas
2. Try move it to the car or another slot
3. It will either not do it or jump to a random
4. You won't be able to use the BZ gas, nor will it move, or it will move to a random slot, or say "You can't do this"

**Expected behavior**
It is expected to move around just like any item in the inventory and it is expected to be able to equip that item and throw it.

**Screenshots**
This often appears too https://media.discordapp.net/attachments/828777058799714334/850098550103605258/unknown.png

**Additional context**
General buggy inventory system when moving BZ gas to and fro between inventories, both player and vehicle.

**Comment by Austin-R418** [comment]
Thanks for reporting the issue. I’m trying to get to the bottom of it. I am currently out of state and away from my computer so I’m going to ask you to get some more information for me.

Can you please put the BZ gas into one of you quick action slots and then press the appropriate keybind for that slot.

**Comment by ItsTheSneakyTurtle** [comment]
> Thanks for reporting the issue. I’m trying to get to the bottom of it. I am currently out of state and away from my computer so I’m going to ask you to get some more information for me.
> 
> Can you please put the BZ gas into one of you quick action slots and then press the appropriate keybind for that slot.

Nothing happens when I do that. My hands remain empty and sometimes it is often difficult to even put it in my inventory with the above-listed issue of "You can't do this action" and inventory slots darting around and switching with things I haven't even tried to switch with.

So, when attempting to equip BZ gas it doesn't do anything except have an empty hand.

**Comment by ItsTheSneakyTurtle** [comment]
Hmmm. For some reason it managed to work lol.

**Comment by ItsTheSneakyTurtle** [comment]
Ok. So what I have found is that the inventory is very buggy for the BZ gas and other things without an image. This sort of stuff happens: https://gyazo.com/e0f59f7dc8e2cc216bb3d53eeb568b7f

However, I have managed to use the BZ gas but if it is stacked it will count 3 BZ gas as one and throw it and you lose however many were stacked. Originally it didn't work so I imagine some random update must have fixed it. I am very surprised. lol
